src/components/Camera/index.js

```diff
@@ -85,11 +85,20 @@ type CameraType = {
   trackScreen: Function,
 }
 
-export default class Camera extends React.Component<CameraType> {
+type CameraStateType = {
+  hasWebcamAccess: boolean,
+  hasSeenPermissionsPrimer: boolean,
+}
 
+export default class Camera extends React.Component<CameraType, CameraStateType> {
   webcam: ?React$ElementRef<typeof Webcam> = null
   interval: ?Visibility
 
+  state:CameraStateType = {
```

Comment:
can state be initialised outside of the constructor in React? Interesting if so.

Reply:
It's part of the stage-2 Babel preset: 

https://babeljs.io/docs/en/babel-plugin-transform-class-properties